xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/pom.xml

```diff
@@ -73,6 +73,13 @@
       <version>${project.version}</version>
       <scope>runtime</scope>
     </dependency>
+    <!-- Used for the translations. -->
+    <dependency>
+      <groupId>org.xwiki.platform</groupId>
+      <artifactId>xwiki-platform-localization-rest-default</artifactId>
+      <version>${project.version}</version>
+      <scope>runtime</scope>
+    </dependency>
```

Comment:
I'm not so sure about this. The live data WebJar doesn't depend on a **specific** implementation of the localization REST API. It just needs an implementation to be present. It doesn't feel right to ask explicitly for the default implementation. @tmortagne WDYT? You have module A that calls a REST URL and module B that implements the REST resource targeted by that URL. Should A depend on B? It doesn't feel right for me.